src/tracks/BBTrack.cpp

```diff
@@ -382,17 +382,10 @@ TrackView * BBTrack::createView( TrackContainerView* tcv )
 
 
 
-TrackContentObject * BBTrack::createTCO( const MidiTime & _pos )
+TrackContentObject* BBTrack::createTCO(const MidiTime & pos)
 {
-	BBTCO * bbtco = new BBTCO( this );
-	if( s_lastTCOColor )
-	{
-		bbtco->setColor( *s_lastTCOColor );
-		bbtco->setUseStyleColor( false );
-	}
-
-	bbtco->movePosition( _pos );
-
+	BBTCO* bbtco = new BBTCO(this);
```

Comment:
What's with this removal?

Reply:
Clarification since I marked the wrong line (oops): this is in reference to the color stuff that was removed.

Reply:
Not sure what removal do you mean, some lines from this method were removed on master due to the Track color PR. I think in the current state of this one there's only the `new BBTCO`, `movePosition` and `return` statements on `BBTrack::createTCO`.

Reply:
Looks like the github diff was acting up, so disregard this 😅